mlflow/utils/logging_utils.py

```diff
@@ -119,7 +121,7 @@ def _configure_mlflow_loggers(root_module_name):
             "loggers": {
                 root_module_name: {
                     "handlers": ["mlflow_handler"],
-                    "level": "INFO",
+                    "level": (MLFLOW_LOGGING_LEVEL.get() or "INFO").upper(),
```

Comment:
`_configure_mlflow_loggers` is called when mlflow is imported. That means, setting `os.environ` after mlflow is imported does nothing.

Reply:
Should we allow modifying it after mlflow is imported?

Reply:
`importlib.reload` can enable debug like this:

```
import importlib
import os

import mlflow


@mlflow.trace
def f():
    pass


f()

print("AFTER".center(80, "-"))


os.environ["MLFLOW_LOGGING_LEVEL"] = "DEBUG"

importlib.reload(mlflow)

f()
```